packages/litexa/src/parser/litexa.pegjs

```diff
@@ -808,14 +809,26 @@ will be included in the response.
 
 RepromptStatement
   = "reprompt" ___ say:SayString {
-      say.reprompt = true;
-      pushSay(location(), say);
-    }
+    say.isReprompt = true;
+    pushSay(location(), say);
+  }
   / "reprompt" {
-      var say = new lib.Say( [new lib.SayEchoPart] );
-      say.reprompt = true;
-      pushSay(location(), say);
-    }
+    const say = new lib.Say( [new lib.SayEchoPart] );
+    say.isReprompt = true;
+    pushSay(location(), say);
+  }
```

Comment:
There was a separate change to remove this syntax since it's relatively useless - could optionally pull that commit into this PR.

Reply:
(from the json localization change, for reference)